.travis.yml

```diff
@@ -27,7 +27,7 @@ env:
 stages:
   - name: Test
   - name: Publish
-    if: ((branch = master AND type = push) OR (tag IS present)) AND NOT fork
+    if: tag IS present AND NOT fork
```

Comment:
@ihostage, I discussed this with @ennru today. We think that it's maybe better to avoid publishing a snapshot on each build to avoid hitting the quota in Bintray.

That said, we only publish releases. If we need to test a snapshot, then we can publish local. If someone in the community needs the lasted merged changes, we cut a release for them. It's now cheap to cut a release, so let's do it on demand.

Reply:
It's was very useful because a release cycle of play-soap was a very long.
But
>It's now cheap to cut a release, so let's do it on demand.

I agree we can try to test this strategy.